src/scicookie/{{cookiecutter.project_slug}}/Makefile

```
@@ -115,6 +115,9 @@ build:
 	hatch build
 {%- elif cookiecutter.build_system == "maturin" %}
 	maturin build	
+{%- elif cookiecutter.build_system == "pybind11" %}
+	python3 -m pip install build
```

```suggestion
	python -m build
```
and add `build` as a dependency in `build-system.requires` in `pyproject.toml`.